WasteWar/Assets/Scripts/Camera Controls/ZoomRouteDraw.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+
+public class ZoomRouteDraw : MonoBehaviour
+{
+    [SerializeField]
+    private Transform[] controlPoints;
+    private Vector3 gizmosPosition;
+
+    private void OnDrawGizmos()
+    {
+        for (float t=0; t <= 1; t += 0.02f)
+        {
+            gizmosPosition = Mathf.Pow(1 - t, 3) * controlPoints[0].position +
```

Comment:
is this the same as the function from Zoom.cs? if so pull it to MathUtils.cs or smth and access that from both